Tests/NIOFileSystemIntegrationTests/FileHandleTests.swift

```diff
@@ -2,7 +2,7 @@
 //
 // This source file is part of the SwiftNIO open source project
 //
-// Copyright (c) 2023 Apple Inc. and the SwiftNIO project authors
+// Copyright (c) 2025 Apple Inc. and the SwiftNIO project authors
```

Comment:
Looks like none of the changes in this file should be necessary, we can use the `NIOFilePath` versions